src/dune_config/display.mli

```diff
@@ -0,0 +1,32 @@
+(** Type of display modes.
+
+    - [status_line] indictates if a status line is shown.
+    - [verbosity] indicates how verbose the display will be. *)
+type t =
+  { status_line : bool
+  ; verbosity : Dune_engine.Display.t
```

Comment:
(Related to my other comment.) It seems sad to depend on the Dune engine just for the sake of this little variant.

Reply:
Yes, agreed. Let's keep the code as loosely coupled as possible. Especially since this case, there's no performance cost.